### DISCUSSION TITLE
我的项目现在支持 fish-speech 1.4 了，效果很好

### DISCUSSION THREAD

**Discussion by zhzLuke96** [fishaudio/fish-speech#674] [Show and tell]
十月一直没空搞，最近终于支持了1.4，测试下来，voice clone效果很强，欢迎各位来试试

这个项目有 api 有 webui，还有音色管理等功能
也在持续开发中，欢迎pr欢迎issues~

项目地址： https://github.com/lenML/Speech-AI-Forge

**Comment by JOEWJ** [comment]
您好 想请教您一下是怎么解决api调用结果是随机人声的呢

**Reply by zhzLuke96** [reply]
你是问我的项目forge的api调用的问题吗？如果是 forge 的api的话你需要指定spk，或者v2 api传递base64的参考音频

如果你问fishspeech项目的api话，应该是需要你的指定 reference_id，并准备好参考音频就行了

**Reply by JOEWJ** [reply]
fishspeech项目的reference_id是官方webui上上传的音频的id吗？那就相当于非完全离线了吗？

**Reply by zhzLuke96** [reply]
> fishspeech项目的reference_id是官方webui上上传的音频的id吗？那就相当于非完全离线了吗？

作者好像回复过你看这个 https://github.com/fishaudio/fish-speech/issues/624

**Reply by JOEWJ** [reply]
我刚发现你回复我了，不好意思。非常感谢
